templates/service-template.yml

```diff
@@ -231,21 +231,11 @@ parameters:
   description: This is the amount of time before a leader lease expires.
   value: "1m"
 
-- name: OBSERVATORIUM_RHSSO_REALM
-  displayName: Observatorium Red Hat SSO realm
-  description: Observatorium Red Hat SSO realm for observability stack.
-  value: ""
-
 - name: OBSERVATORIUM_RHSSO_TENANT
   displayName: Observatorium Red Hat SSO tenant
   description: Observatorium Red Hat SSO tenant for observability stack.
   value: ""
 
-- name: OBSERVATORIUM_RHSSO_AUTH_SERVER_URL
```

Comment:
same comment as https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1682/files#r1153062704

Reply:
See the additional remarks in the original description